src/crimson/os/seastore/cache.h

```diff
@@ -562,8 +562,8 @@ class Cache {
 
   using src_ext_t = std::pair<Transaction::src_t, extent_types_t>;
   struct {
-    std::unordered_map<Transaction::src_t, uint64_t> trans_created;
-    std::unordered_map<Transaction::src_t, uint64_t> trans_committed;
+    std::vector<uint64_t> trans_created_by_src;
+    std::vector<uint64_t> trans_committed_by_src;
```

Comment:
could have simple helper methods here for indexing these vectors using `src_t`.

Reply:
introduced `get_counter()` helper.